app/views/recipes/public.html.erb

```diff
@@ -16,8 +12,8 @@
     
   </div>
     <p class='col'>
-    food items:  <%=recipe.recipefoods.length%><br>
-    total price:  <%=@sum%>
+    food items: <br>
+    total price:  
     </p>
```

Comment:
- > on the public recipes you are displaying the food item and total price but which has no corresponding values please implement that part to make your project complete